data/json/items/book/speech.json

```diff
@@ -52,5 +52,24 @@
     "max_level": 6,
     "intelligence": 8,
     "time": 30
+  },
+  {
+    "id": "dnd_handbook",
```

Comment:
Surely this should spawn in-world too.

Reply:
I was initially against it, as it doesn't train skills that existing textbooks don't do better already, and intended it as a profession book ala Hitchhiker's Guide to the Cataclysm. If you want, I can put it into the loot tables.

Reply:
My inner nerd would really like that. Perhaps as a rare spawn in houses and maybe coffee shops?

Reply:
I might sprinkle it into the recently-added LAN centers as well, I feel like it'd fit there.